src/main/resources/view/DarkTheme.css

```diff
@@ -132,6 +132,15 @@
     -fx-text-fill: #010504;
 }
 
+.cell_request_label {
```

Comment:
So the request is like a tag?

Reply:
yep, for now it looks like the tag and undelivered tag, wanted to make it stand out more.